services-custom/dynamodb-enhanced/README.md

```diff
@@ -64,18 +64,18 @@ values used are also completely arbitrary.
        .build();
    ```
    
-3. Create a MappedDatabase object that you will use to repeatedly
+3. Create a DynamoDbEnhancedClient object that you will use to repeatedly
    execute operations against all your tables :- 
    ```java
-   MappedDatabase database = DynamoDbMappedDatabase.builder()
-                                                   .dynamoDbClient(dynamoDbClient)
-                                                   .build();
+   DynamoDbEnhancedClient enhancedClient = DefaultDynamoDbEnhancedClient.builder()
```

Comment:
There should be no open reference to Default*. ```DynamoDbEnhancedClient.builder()``` needs to work, if it doesn't we need to fix that too.

Reply:
Yes, I did a direct name translation of these statements without switching over to the new paradigm. Will fix